sync2jira/downstream_issue.py

```diff
@@ -17,8 +17,9 @@
 #
 # Authors:  Ralph Bean <rbean@redhat.com>
 
-# Python Standard Library Modules
 from datetime import datetime, timezone
+
+# Python Standard Library Modules
 import difflib
 import logging
 import operator
```

Comment:
These imports should all be in one block (i.e., `datetime` _is_ a Standard Library module), so we should probably just remove the comment at line 22 (and line 29 and line 36) -- now that we have `isort`, these blocks will be maintained automatically, no there is no need to label them.